doc/docs/Python_User_Interface.md

```diff
@@ -1158,7337 +1335,6462 @@ If `match_frequency=False` or `kpoint` is not zero in the given `direction`, the
 
+As `load_energy`, but negates the Fourier-transformed fields after they are
+loaded. This means that they will be *subtracted* from any future field Fourier
+transforms that are accumulated.
 
+</div>
 
 </div>
 
-<a id="Simulation.get_hfield_x"></a>
 
-**Simulation.get\_hfield\_x**
+### Force Spectra
 
-```python
-def get_hfield_x(self):
-```
+Very similar to flux spectra, you can also compute **force spectra**: forces on an object as a function of frequency, computed by Fourier transforming the fields and integrating the vacuum [Maxwell stress tensor](https://en.wikipedia.org/wiki/Maxwell_stress_tensor):
 
-<div class="method_docstring" markdown="1">
+$$\sigma_{ij} = E_i^*E_j + H_i^*H_j - \frac{1}{2} δ_{ij} \left( |\mathbf{E}|^2 + |\mathbf{H}|^2 \right)$$
 
+over a surface $S$ via $\mathbf{F} = \int_S \sigma d\mathbf{A}$. You should normally **only evaluate the stress tensor over a surface lying in vacuum**, as the interpretation and definition of the stress tensor in arbitrary media is often problematic (the subject of extensive and controversial literature). It is fine if the surface *encloses* an object made of arbitrary materials, as long as the surface itself is in vacuum.
 
+See also [Tutorial/Optical Forces](Python_Tutorials/Optical_Forces.md).
 
-</div>
+Most commonly, you will want to **normalize** the force spectrum in some way, just as for flux spectra. Most simply, you could divide two different force spectra to compute the ratio of forces on two objects. Often, you will divide a force spectrum by a flux spectrum, to divide the force $F$ by the incident power $P$ on an object, in order to compute the useful dimensionless ratio $Fc$/$P$ where $c=1$ in Meep units. For example, it is a simple exercise to show that the force $F$ on a perfectly reflecting mirror with normal-incident power $P$ satisfies $Fc$/$P=2$, and for a perfectly absorbing (black) surface $Fc$/$P=1$.
 
-<a id="Simulation.get_hfield_y"></a>
+The usage is similar to the [flux spectra](Python_Tutorials/Basics.md#transmittance-spectrum-of-a-waveguide-bend): you define a set of [`ForceRegion`](#ForceRegion) objects telling Meep where it should compute the Fourier-transformed fields and stress tensors, and call `add_force` to add these regions to the current simulation over a specified frequency bandwidth, and then use `display_forces` to display the force spectra at the end. There are also `save_force`, `load_force`, and `load_minus_force` functions that you can use to subtract the fields from two simulation, e.g. in order to compute just the force from scattered fields, similar to the flux spectra. The function used to add a [`ForceRegion`](#ForceRegion) object is defined as follows:
 
-**Simulation.get\_hfield\_y**
+
+<a id="Simulation.add_force"></a>
+
+<div class="class_members" markdown="1">
 
 ```python
-def get_hfield_y(self):
+def add_force(self, *args):
 ```
 
 <div class="method_docstring" markdown="1">
 
+add_force(fcen, df, nfreq, freq, ForceRegions...)
```

Comment:
Same issue here: `add_force(fcen, df, nfreq, freq, ForceRegions...)` should be a code block rather than part of the prose description?